apps/ftu/style/style.css

```diff
@@ -816,6 +816,10 @@ aside.connecting {
 #fb_after_import1 {
   background: url('images/import_fb.png') no-repeat center left / 3rem;
 }
+html[dir="rtl"] #fb_after_import1 {
+  background-position: right center;
+}
+
```

Comment:
not sure if it should be in here next to the affected tag or on the `RTL Tweaks` section (around line 1200)